charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -80,6 +80,9 @@ spec:
         {{- if .Values.namespacesDenylist }}
         - --namespaces-denylist={{ tpl (.Values.namespacesDenylist | join ",") $ }}
         {{- end }}
+        {{- if .Values.releaseNamespace }}
```

Comment:
Can you move this a couple of lines up, so it is below the other item that configures namespaces.

Also: what happens if both are set?

Reply:
> Can you move this a couple of lines up, so it is below the other item that configures namespaces.

Done

> Also: what happens if both are set?

Good point. I've updated the template and comments. Have another look pls